docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,90 @@
+---
+id: slack
+title: Slack connector
+description: Working with Slack connector
+---
+
+The **Slack connector** allows you to interact with a [Slack](https://slack.com/) workspace, for example by sending Slack messages from your BPMN processes.
+
+## Prerequisites
+
+To use the **Slack connector** the following prerequisites need to be set-up. Read the [appendix](#appendix) to find out more.
```

Comment:
For existing documentation we tried to clearly gear towards the following personas (structuring sections accordingly):

* Process Developer (connector user)
* Camunda Administrator (IT admin, connector administrator)

The prerequisite section acts as a friendly reminder that prior setup is required for Connector Administrators but explicitly does not go into detail what that setup comprises. In particular the only relevant bit that a connector user needs to know is that she must use a OAuth token (provided by  / setup by the administrator, cf. appendix). We detail that in the "using connector" / "Create a Slack connector task" section.

Could we attempt the same here? Something along the lines of 

> ## Prerequisites
> 
> To use the slack connector a slack app must be registered with the slack workspace you would like to send messages to. Read the appendix to on how to set it up.

Reply:
Yes, makes sense. I changed accordingly.